src/applications/facility-locator/components/ServicesAtFacility.jsx

```diff
@@ -91,13 +90,28 @@ class ServicesAtFacility extends Component {
       },
     } = this.props;
 
-    if (!services.benefits || isEmpty(services.benefits.standard)) {
+    if (
+      !services.benefits ||
+      services.benefits.length === 0 ||
+      (services.benefits.standard && services.benefits.standard.length === 0)
```

Comment:
You can shorten this to:

```js
if (!services?.benefits?.standard) {
  return null;
}
```

BTW really like how you're escaping early here, makes it easy to read 💯